src/libsystemd-network/fuzz-dhcp-server.c

```diff
@@ -54,7 +54,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
         struct in_addr address = { .s_addr = htobe32(UINT32_C(10) << 24 | UINT32_C(1))};
         _cleanup_free_ uint8_t *duped = NULL;
 
-        if (size < sizeof(DHCPMessage))
+        if (size < sizeof(DHCPMessage) || size > 600)
```

Comment:
From the .options file. It was added thus in the initial commit in 2bd37c5be2f09950f26351506a00929c8719ea15.

Reply:
I don't know why it was added there either. But since the file have never affected AFL and honggfuzz it effectively has been fuzzed without any arbitrary limits. As far as I can remember the dns packets were limited somewhere for no apparent reason as well.